packages/sdk/src/cross-chain-messenger.ts

```diff
@@ -1868,13 +1868,28 @@ export class CrossChainMessenger {
         recipient?: AddressLike
         l2GasLimit?: NumberLike
         overrides?: PayableOverrides
-      }
+      },
+      isEstimatingGas: boolean = false
     ): Promise<TransactionRequest> => {
+      const getOpts = async () => {
+        if (isEstimatingGas) {
+          return opts
+        }
+        const gasEstimation = await this.estimateGas.depositETH(amount, opts)
+        gasEstimation.mul(1.5)
```

Comment:
you need to assign here to `gasEstimation`, it doesnt mutate, it copies

Reply:
holy my brain isn't working wtf how'd I read this without noticing it?

Reply:
Oh I see actually this is right I just have a line of code I didn't delete.  I do this on the next line this line of code is just dead code